src/analyze/analyze-security.c

```diff
@@ -1709,6 +1711,10 @@ static int assess(const SecurityInfo *info, Table *overview_table, AnalyzeSecuri
                         return table_log_add_error(r);
         }
 
+        /* Return error when overall exposure level is over MEDIUM */
+        if (exposure > 50 && FLAGS_SET(flags, ANALYZE_SECURITY_ERROR_IF_EXPOSURE_ABOVE_MEDIUM))
+                return -EINVAL;
```

Comment:
oh, i am not sure i agree with this, this sounds pretty arbitrary.

at the very least it shouldn't be snuck in with an unrelated commit like this one.

Reply:
This only applies when running with the --offline switch though, so it is related? Wanted to have it configurable as a follow-up btw